### DISCUSSION TITLE
Remove -alpha.0 from versions?

### DISCUSSION THREAD

**Discussion by korywka** [MasterKale/SimpleWebAuthn#269] [General]
<img width="125" alt="image" src="https://user-images.githubusercontent.com/988471/190451490-c19cb658-b57a-4c8f-8f01-31471b2bc83f.png">

It looks like this package is still in alpha but it is not.

**Comment by MasterKale** [comment]
6.3.0 is definitely in alpha, it's a preview build containing a WIP implementation of Device Public Key support. It's not intended for use in production environments; everyone else should still be on 6.2.0.

Is an `npm install` or `npm upgrade` pushing 6.3.0-alpha.0 on you?

**Reply by korywka** [reply]
Oh thanks. I thought it was added by lerna by mistake.

Sorry to be a distraction.

**Comment by lkwr** [comment]
It seems this is still a problem. When looking at the [@simplewebauthn/typescript-type](https://www.npmjs.com/package/@simplewebauthn/typescript-types) package at npm, it shows the version `6.3.0-alpha.1` but I should expect to see the latest stable release (in this case `6.2.1`): 

<img width="434" alt="Screenshot 2023-01-02 at 16 37 08" src="https://user-images.githubusercontent.com/49527534/210252592-b3bbaa83-205e-4437-8e33-6f424a98c2fe.png">

This can be a bit confusing as you may think the whole package is "alpha". I would recommend that pre-released versions (such as alpha, betas and so) should NOT be tagged as `latest` in the npm registry. This solves the problem of showing pre-released versions on the default package page.

**Reply by MasterKale** [reply]
Huh, interesting, browser and server both show 6.2.2:

![Screenshot 2023-01-02 at 10 16 57 AM](https://user-images.githubusercontent.com/5166470/210266961-b8c2c85f-7421-419c-aefe-abedc62e3b57.png)
![Screenshot 2023-01-02 at 10 17 03 AM](https://user-images.githubusercontent.com/5166470/210266962-2e367a23-3b61-414b-a23d-abe7e43b32ed.png)

I suppose what happened here is that 6.2.2 didn't include any changes to **typescript-types** and so it's still showing the alpha release.

Thank you for pointing out that `latest` is what determines the version that shows up there. I use `lerna publish` to get things up on NPM, perhaps there's a flag there I need to use when pushing up alpha launches. I'll take a look next time I need to do so.

**Reply by lkwr** [reply]
Maybe something like [this (`--dist-tag`)](https://github.com/lerna/lerna/tree/main/commands/publish#--dist-tag-tag) or [that (`--canary`)](https://github.com/lerna/lerna/tree/main/commands/publish#--canary) should do the trick with `lerna publish`.
